tests/schemas/api_v3/dryrunTransaction.schema.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+
+const regex = require('./regex');
+
+const eventItem = {
```

Comment:
Can we reuse the `event` schema here? Also, `data` ideally should be an object.

Reply:
The structure in these two differs.